build.gradle

```diff
@@ -72,6 +72,14 @@ allprojects {
 				includeGroup "com.jamieswhiteshirt"
 			}
 		}
+		maven {
```

Comment:
![image](https://github.com/user-attachments/assets/ff3cc14a-0c32-40e1-b018-a96afcc86fb6)

Reply:
I suppose this is leftovers from including geckolib?